pkg/kubelet/stats/cadvisor_stats_provider.go

```diff
@@ -192,6 +206,21 @@ func isPodManagedContainer(cinfo *cadvisorapiv2.ContainerInfo) bool {
 	return managed
 }
 
+// getcadvisorPodInfoFromPodUID returns a pod cgroup information by matching the podUID with its CgroupName identifier
+// In ContainerInfo data struct, the key for pod cgroup is one of the following format (<podUID> is the string of the pod UID)
+// "/.../kubepods/pod<podUID>"
```

Comment:
you can remove these comments, since we are only relying on the base.

Reply:
sure, but I was thinking to put it here so it is easy to understand for cgroup name path?